src/pages/storage/StorageVolumeNameLink.tsx

```diff
@@ -12,43 +12,53 @@ interface Props {
   className?: string;
 }
 
-export const generateLinkForVolumeDetail = (args: {
-  volume: LxdStorageVolume;
-  project: string;
-}) => {
-  const { volume, project } = args;
-  let path = `storage/pool/${volume.pool}/volumes/${volume.type}/${volume.name}`;
-
+export const generateLinkForVolumeDetail = (
```

Comment:
I think `generateLinkForVolumeDetail` and `hasDetailPage` should be moved to `util/storageVolume.tsx`?